ui/src/commonMain/kotlin/com/materiiapps/gloom/ui/components/LargeSegmentedButton.kt

```diff
@@ -26,15 +27,19 @@ import androidx.compose.ui.unit.dp
 fun RowScope.LargeSegmentedButton(
     icon: Any,
     iconDescription: String? = null,
-    text: String
+    text: String,
+    onClick: () -> Unit = {},
+    enabled: Boolean = true
 ) {
     Column(
         horizontalAlignment = Alignment.CenterHorizontally,
         verticalArrangement = Arrangement.spacedBy(12.dp, Alignment.CenterVertically),
         modifier = Modifier
+            .clickable(enabled) { onClick() }
```

Comment:
This is required to let Compose know this is a button for a11y:
```suggestion
            .clickable(enabled = enabled, role = Role.Button) { onClick() }
```
Note: Requires import `androidx.compose.ui.semantics.Role`